modules/rgbd/src/submap.hpp

```diff
@@ -119,6 +121,24 @@ void Submap<MatType>::updatePyrPointsNormals(const int pyramidLevels)
     buildPyramidPointsNormals(points, normals, pyrPoints, pyrNormals, pyramidLevels);
 }
 
+template<typename MatType>
+void Submap<MatType>::addEdgeToSubmap(const int tarSubmapID, const Affine3f& tarPose )
+{
+    // duplicate check.
+    auto& iter = constraints.find(tarSubmapID);
+
+    // if there is NO edge of currSubmap to tarSubmap.
+    if(iter == constraints.end())
+    {
+        // Frome pose to tarPose transformation
+        Affine3f estimatePose = tarPose * pose.inv();
+
+        // Create new Edge.
+        PoseConstraint& preConstrain = getConstraint(tarSubmapID);
+        preConstrain->accumulatePose(estimatePose, 1);
+    }
+}
```

Comment:
Hi, @savuor. When the LOOP is detected, I need to add new edges and constraints. Is this implementation right?